src/queries/label-text.js

```diff
@@ -51,15 +36,38 @@ function queryAllLabelsByText(
     .map(({node}) => node)
 }
 
-function getLabelContent(label) {
-  let labelContent = label.getAttribute('value') || label.textContent
-  Array.from(label.querySelectorAll('textarea')).forEach(textarea => {
-    labelContent = labelContent.replace(textarea.value, '')
-  })
-  Array.from(label.querySelectorAll('select')).forEach(select => {
-    labelContent = labelContent.replace(select.textContent, '')
-  })
-  return labelContent
+function getTextContent(node) {
+  const TEXT_NODE = 3
+  const labelledNodeNames = [
+    'button',
+    'meter',
+    'output',
+    'progress',
+    'select',
+    'textarea',
+    'input',
+  ]
+  if (
+    labelledNodeNames.some(nodeName => node.nodeName.toLowerCase() === nodeName)
+  ) {
+    return ''
```

Comment:
Just a silly comment, could maybe be more clear to write the if condition like this:
`labelledNodeNames.includes(node.nodeName.toLowerCase())`?

Reply:
yes, thanks :)